### ISSUE TITLE
[Mapping issue] for future mother-baby linkage

### ISSUE THREAD

**Issue by tiozab** [oxford-pharmacoepi/cdm-etl-gold#29]
Not a problem, but a suggestion:
Jamie's algorithm to identify mother-baby linkage uses the family_source_value. 
The CDM data model stores this number in the ppp table (payer_plan_period). 
If we want to establish mother-baby-links in the future, we would need this number in the omopped data of CPRD (the ppp table is currently empty). The number exists in CPRD GOLD and is called "family number" or "famnum". https://cprd.com/sites/default/files/2022-02/CPRD%20GOLD%20Full%20Data%20Specification%20v2.5.pdf
Basically, we would need the ppp table populated with person_id, family_id and family_source_value (and a unique ppp id, e.g. a row number) @jamieweaver @daniprietoalhambra

**Comment by adelmestri** [comment]
@tiozab, @jamieweaver, @daniprietoalhambra : Thank you, it's a good suggestion. 

Unfortunately, the "famnum" is not a family number, even if the name might suggest so, but a patient’s home address at registration and cannot be used safely to identify mother-baby link. It is used for any shared accomodation or building address. Also, famnum is not unique across all practices. This is the reason why CPRD have created a separate mother-baby linkage dataset. 

However, @daniprietoalhambra told me that we do not have any projects requiring this or paying for the linkage or for the mapping of this. So it is not a priority at the moment. We might look into this in due course.
